sal/class.json-api-site-jetpack-base.php

```diff
@@ -97,7 +97,7 @@ function get_jetpack_modules() {
 	}
 
 	function is_vip() {
-		return false; // this may change for VIP Go sites, which sync using Jetpack
+		return defined( 'WPCOM_IS_VIP_ENV' ) ? (bool) WPCOM_IS_VIP_ENV : false;
```

Comment:
Can this just be `return defined( 'WPCOM_IS_VIP_ENV' ) && true === WPCOM_IS_VIP_ENV;`? We tend to make these kinds of checks explicit to a `true` value, and `false` for anything else.

Reply:
Makes sense. I'll change that.